metadata.json

```diff
@@ -28,7 +28,7 @@
   "requirements": [
     {
       "name": "puppet",
-      "version_requirement": ">= 5.5.8 < 7.0.0"
+      "version_requirement": ">= 6.0.0 < 8.0.0"
```

Comment:
6.0.0 had issues related to systemd service management so we trend to set the lower bound to `>= 6.1.0`.

Reply:
Ah, okay, I've upped the minimal Puppet version to 6.1.0.